tensorflow/training/docker/1.15.3/py36/Dockerfile.cpu

```diff
@@ -1,6 +1,7 @@
 FROM ubuntu:18.04
 
 LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="4"
```

Comment:
The training py37 TF 1.15.2 image in prod repo is with version 7.0

Reply:
Valid point. Will make the change.